decidim-admin/app/commands/decidim/admin/reorder_participatory_process_steps.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # A command that sets a step in a participatory process as active (and
+    # unsets a previous active step)
+    class ReorderParticipatoryProcessSteps < Rectify::Command
+      # Public: Initializes the command.
+      #
+      # collection - an ActiveRecord::Relation of steps
+      # order_string - a String representing an Array holding the order of IDs
```

Comment:
I didn't want to rescue a JSON parsing in the controller but yeah, feels weird to me too. Changing it!